### PR TITLE
Feature/artist page

### PR DESCRIPTION
**Pull request by onebarloop**
I finished my 2nd User Story. You can check it [here](https://github.com/onebarloop/capstone-project/issues/2)

I'm very happy about feedback!

### CODE REVIEW

File: components/Artist.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  location,
+  tattoos,
+}: ArtistProps): JSX.Element {
+  return (
+    <StyledMain>
+      <StyledInfoCard>
+        Information
+        <li>
+          Artist:
+          <br /> {artistName}
```
Comment (stefankorder) [inline components/Artist.tsx:25]: Try not to use `<br />`, use CSS properties or other tags to break the text. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br?retiredLocale=de#accessibility_concerns

File: components/Artist.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        </li>
+        <li>
+          Location: <br />
```
Comment (stefankorder) [inline components/Artist.tsx:28]: Try not to use `<br />`, use CSS properties or other tags to break the text.

**Review by stefankorder — APPROVED** [review]
LGTM! Just a little hint. Take a look at my comments.

File: components/Artist.tsx (hunk 2 above)
Comment (onebarloop) [reply components/Artist.tsx:28]: Thanks for the review! Will look for a css-way to handle the line break.

**Pull request merged**